### ISSUE TITLE
aria-label is missing in the page sections where hasOverflowScroll is used. 

### ISSUE THREAD

**Issue by mmelko** [hawtio/hawtio-next#900]
```
VM411081 PageSection.js:56 PageSection: An accessible aria-label is required when hasOverflowScroll is set to true. 
    at PageSection (webpack://app/../node_modules/@patternfly/react-core/dist/esm/components/Page/PageSection.js?:51:11)
    at div
    at PageGroup (webpack://app/../node_modules/@patternfly/react-core/dist/esm/components/Page/PageGroup.js?:19:11)
    at CamelContent (webpack://app/../packages/hawtio/dist/index.js?:12496:58)
    at div
    at div
    at SplitWrapper (webpack://app/../node_modules/react-split/dist/react-split.es.js?:18:20)
    at Camel (webpack://app/../packages/hawtio/dist/index.js?:12697:59)
    at RenderedRoute (webpack://app/../node_modules/react-router/dist/index.js?:577:5)
    at Routes (webpack://app/../node_modules/react-router/dist/index.js?:1267:5)
    at main
    at div
    at Page (webpack://app/../node_modules/@patternfly/react-core/dist/esm/components/Page/Page.js?:38:9)
    at HawtioPage (webpack://app/../packages/hawtio/dist/index.js?:18803:58)
    at RenderedRoute (webpack://app/../node_modules/react-router/dist/index.js?:577:5)
    at Routes (webpack://app/../node_modules/react-router/dist/index.js?:1267:5)
    at Router (webpack://app/../node_modules/react-router/dist/index.js?:1201:15)
    at BrowserRouter (webpack://app/../node_modules/react-router-dom/dist/index.js?:693:5)
    at Hawtio (webpack://app/../packages/hawtio/dist/index.js?:18858:11)

```

**Comment by tadayosi** [comment]
@mmelko Maybe next time, try not to use the commit message with format `fix(#xxx): ...` as it doesn't automatically close the mentioned issue. Use `fix(<component>): ...\n\nFix #xxx` instead.
